app/helpers/format-text.js

```diff
@@ -12,16 +14,24 @@ export function formatText([ text ], hash) {
 
   let allowedTags = [ 'strong', 'em', 'b', 'i', 'code', 'pre', 'sub', 'sup', 'br' ];
   let sanitizedText = SanitizeHtml(text, { allowedTags });
-  let words = sanitizedText.split(' ');
-  let len = hash.limit || 500;
-  let out = '';
 
-  if (words) {
-    out = words.splice(0, len).join(' ');
+  let out = '';
 
-    if (words.length > len) {
-      out += ' …';
+  // Doesn't parse math correctly (so leave it out for now).
+  if (!checkForTex(sanitizedText)) {
+    let words = sanitizedText.split(' ');
+    let len = hash.limit || 500;
+  
+    if (words) {
+      out = words.splice(0, len).join(' ');
+  
+      if (words.length > len) {
+        out += ' …';
+      }
     }
+
+  } else {
+    out = sanitizedText;
```

Comment:
Will this cause issues visual issues if the DoiCitationBody is too long? Is that a negligible concern?

Reply:
It might.  However, to work correctly, that would require us to be able to parse the mathematical encoding, otherwise we might end up with something like https://search.datacite.org/works?query=10.17182%2Fhepdata.130967.v1.  That was summarized by cutting off the body and adding '...' at the end, but you will notice that it doesn't render the last mathematics.  So it was a choice between having the summary be longer than desired, or having an error in the rendered notation.  I left that as something we can do in the future, and should document it.

I did check to see if there was a solution, but could find nothing.  I will take a look around again though.